frontend/src/components/Board/Board.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+import {
+  BoardInfoContainer,
+  BoardInfo,
+  BoardImage,
+  BoardTitle,
+  AuthorInfoContainer,
+  AuthorName,
+  AuthorAvatar
+} from "./Board.styled";
+import DotsButton from "../DotsButton/DotsButton";
```

Comment:
Mógłbyś dorzucić fotkę tego komponentu? 
Może on wyglądać różnie:

![image](https://user-images.githubusercontent.com/10167978/150803369-c89c22d0-719b-4e8a-823b-ef24fe36d2a4.png)

Reply:
<img width="924" alt="Zrzut ekranu 2022-01-24 o 23 11 06" src="https://user-images.githubusercontent.com/27691628/150873611-9c56a73f-c9c3-4b77-a047-f0dcb3883dbe.png">
<img width="922" alt="Zrzut ekranu 2022-01-24 o 23 11 22" src="https://user-images.githubusercontent.com/27691628/150873615-5946fc63-2576-4dfd-92b6-ba1bb27e193e.png">

Reply:
Ale to do całego PR, tutaj nikt tego nie zobaczy :D